client/my-sites/email/form/mailboxes/types.ts

```diff
@@ -10,12 +11,42 @@ enum EmailProvider {
 
 interface MailboxFormField< Type > {
 	error: FieldError;
+	isRequired: boolean;
+	isVisible: boolean;
+	readonly typeName: string;
 	value: Type;
 }
 
 abstract class MailboxFormFieldBase< T > implements MailboxFormField< T > {
-	error: FieldError = null;
+	private _error: FieldError = null;
```

Comment:
Why do we prefix this class attribute with an underscore? Is it a convention we have defined somewhere in our documentation?